### PR TITLE
 [sailfish-utilities] Add a Restart Fingerprint button. Fixes JB#57941

### PR DESCRIPTION
**Pull request by llewelld**
Adds a button for restarting the fingerprint service. In practice, it
restarts, waits 10 seconds and then restarts again.

In some situations where the fingerprint cache gets messed up, which
prevents fingerprints from being recognised, this sequence resolves the
issue.

### CODE REVIEW

**Comment by llewelld** [comment]
See the related forum thread for background: https://forum.sailfishos.org/t/fingerprint-record-lost-randomly/6283/81

File: qml/plugins/RestartFingerprint.qml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import QtQuick 2.0
+import Sailfish.Silica 1.0
+import Sailfish.Utilities 1.0
+
+ActionItem {
+    //% "Fingerprint"
+    title: qsTrId("sailfish-tools-he-fingerprint")
+    //% "Restart"
+    actionName: qsTrId("sailfish-tools-bt-finterprint_restart")
```
[the remark this answers is not in the document]
Comment (llewelld) [reply qml/plugins/RestartFingerprint.qml]: Nicely spotted, thank you. My fingers seem to insist on typing finterpring :(
This one is fixed now, at any rate.

**Comment by llewelld** [comment]
> Does it need a whole ten seconds? That'll seem like a long time with a spinner.

Yes, it is a long time to wait, you're right. I basically took this from the [reports on the forum](https://forum.sailfishos.org/t/fingerprint-record-lost-randomly/6283/62 ) which all seem to use either 10 or 20 seconds as the gap. But it would be better if it could be less.

We could go with something short and increase the length if it turns out to be unreliable. Would you have a suggestion as a starting point?

**Comment by abranson** [comment]
I think I usually only wait for 2 or 3, so 5 might be enough

**Comment by attah** [comment]
I too think 10 is excessive. I run the restart commands basically as fast as i can type my password.

**Comment by llewelld** [comment]
Thanks for the feedback both. I've set it to 5 seconds. It still feels like quite a long time, so maybe it could go lower?

**Comment by abranson** [comment]
Maybe two or three? You can always press it again if it wasn't long enough.

**Review by abranson — APPROVED** [review]

**Comment by llewelld** [comment]
I've set it to three. If it proves ineffective, we can bump it up later (it's pretty easy to change on-device as well).

**Comment by llewelld** [comment]
Updated commit message s/10/3/g

**Pull request merged**